tests/test_logging.py

```diff
@@ -14,6 +19,19 @@ def has_handler(logger, handler_name):
     return any([handler.name == handler_name for handler in logger.handlers])
 
 
+def capture_stdout(command, *args, **kwargs):
```

Comment:
There's https://docs.python.org/3.3/library/test.html#test.support.captured_stderras as of Python 3.something (captured_stdout is available on 2.7) and also Django's vendored version at https://github.com/django/django/blob/master/django/test/utils.py#L553 as of 1.8.